binaryscalarexpr.go

```diff
@@ -111,7 +111,13 @@ func Min(chunk parquet.ColumnChunk) parquet.Value {
 	columnIndex := chunk.ColumnIndex()
 	min := columnIndex.MinValue(0)
 	for i := 1; i < columnIndex.NumPages(); i++ {
-		if v := columnIndex.MinValue(i); compare(min, v) == 1 {
+		v := columnIndex.MinValue(i)
+		if min.IsNull() {
+			min = v
+			continue
```

Comment:
nothing is less than `NULL` I think, so we could just return here as long as the ordering is ascending.